miden-remote-provers/src/lib.rs

```diff
@@ -4,40 +4,37 @@ use alloc::string::String;
 
 pub mod generated;
```

Comment:
Do we not need to gate this with `tx-prover` feature?

Reply:
Should this definition be within the `prover` module? If nothing here uses it then it might be cleaner to leave it within that module

Reply:
Maybe we could move `RemoteProverError` as well? Or is there something else that could use the error without the feature?

Reply:
>Should this definition be within the prover module? If nothing here uses it then it might be cleaner to leave it within that module

Agree, I'm motiving it there.